docs/commandline_interface.rst

```diff
@@ -0,0 +1,54 @@
+Command-line interface
```

Comment:
I think this file should actually be moved to be more in-line with other documentation:
`docs/commandline_interface.rst` -> `docs/user/cli.rst`
This will also require a few changes, including adding `.. _cli:` to the start of the file, and `docs/index.rst` will need to have the file added to its User Guide section

Reply:
I just put it there, because I had no better idea, where to put it
I will move it in user - do you think, that all users know, what  '_CLI_' means (I tend to  avoid abbreviations for clarity) ? 
(since there will  link in the documentation it might be not too critical)